client/notifications/src/panel/boot/stylesheets/main.scss

```diff
@@ -504,8 +504,8 @@
 		display: flex;
 		flex-direction: column;
 		height: 100%;
-		left: 0px;
-		right: 0px;
+		left: 0;
+		right: 0;
 		box-shadow: -3px 1px 10px -2px rgba( var( --color-neutral-700-rgb ), 0.075 );
 
 		@media only screen and ( min-width: 480px ) {
```

Comment:
Sidenote: it would be good to lint for these and suggest to replace with the breakpoint mixin